modules/custom/d_blog/d_blog.module

```diff
@@ -23,7 +23,7 @@ function d_blog_theme($existing, $type, $theme, $path) {
     'paragraph__d_p_blog_image' => [
       'base hook' => 'paragraph',
     ],
-    'field__node__field_blog_main_image__blog_post' => [
+    'field__node__field_blog_media_main_image__blog_post' => [
```

Comment:
Czy to musi być w hook_theme? Nie wystarczy dodać theme suggestions?

Reply:
a dodanie theme suggestions sprawi, ze zaczyta twiga z modulu? bo te sugestie już są i bez tego hook_theme :point_up: